### DISCUSSION TITLE
Analysis of photos stopped at some point

### DISCUSSION THREAD

**Discussion by SebWo87** [matiasdelellis/facerecognition#740] [Q&A] [answered]
Hi everyone! 

I ran facerecognition on my photo collection, but at some point in the analysis phase the process stopped.

occ face:progress shows below info since several weeks. Interesting fact here is, that new photos seems to be processed. Sometimes I see the "Remaining" raising by a small amount, but it stops every time at 1339.
+----------+---------------+--------------+
| Images | Remaining | ETA          |
+----------+---------------+--------------+
| 174964 | 1339          | in 5 hours |
+----------+---------------+--------------+

Any ideas where I can find logs or more info about this?

**Comment by matiasdelellis** [comment]
Hi @SebWo87 
You should run the face:background job command manually with greater debugging and see what happens.

Add '-vvv' to the command line

**Comment by SebWo87** [comment]
Thanks for the hint @matiasdelellis !
I get the following response:
```
1/8 - Executing task CheckRequirementsTask (Check all requirements)
	System: Linux
	System memory: 7769649152
	PHP Memory Limit: 4294967296
	Clustering backend: PHP (Not recommended.
	Image Backend: Imaginary
	An Imaginary service (http://nextcloud-aio-imaginary:9000) was configured to manage temporary images, but it is inaccessible.Check out the service, or set the 'preview_imaginary_url' key appropriately.
Task CheckRequirementsTask signalled we should not continue, bailing out
```
Since I run Nextcloud within the AIO docker environment, it seems to be more complicated to get this to work. 
I tried reaching the imaginary container from within the main nextcloud container and this succeeded. So unfortunately I don't see any problem in connecting to imaginary :(

**Comment by SebWo87** [comment, marked as answer]
Seems to be solved with  ![Add key support for Imaginary #746](https://github.com/matiasdelellis/facerecognition/pull/746)! Thank you all for your support!
